ethcore/src/types/trace_types/filter.rs

```diff
@@ -127,7 +132,7 @@ impl Filter {
 		};
 
 		action || match trace.result {
-			Res::Create(ref create) => self.to_address.matches(&create.address),
+			Res::Create(ref create) => self.to_address.stricly_matches(&create.address),
```

Comment:
However the `address` field is in the `result` _part_ of the trace, not in the action